fighthealthinsurance/static/js/scrub_scrub.ts

```
@@ -2,62 +1,105 @@
     }
-    for (var i=0; i < reservedTokens.length; i++) {
-	text = text.replace(
-	    reservedTokens[i][0],
-	    " " + reservedTokens[i][1]);
+  }
+  console.log("Preparing to scrub:");
+  console.log(text);
+  console.log("Scrubbing with:");
+  console.log(reservedTokens);
+  console.log(scrubRegex);
+  for (var i = 0; i < scrubRegex.length; i++) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)